firefox-ios/Client/Frontend/TrackingProtection/CertificatesViewController.swift

```diff
@@ -0,0 +1,276 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Common
+import Shared
+import SiteImageView
+
+import Security
+import CryptoKit
+import X509
+import SwiftASN1
+
+struct CertificateKeys {
+    static let commonName = "CN"
+    static let country = "C"
+    static let organization = "O"
+}
+
+struct CertificatesUX {
+    static let sectionLabelWidth = 150.0
+    static let sectionLabelMargin = 20.0
+    static let sectionItemsSpacing = 40.0
+    static let allSectionItemsSpacing = 10.0
+    static let allSectionItemsTopMargin = 20.0
+    static let headerStackViewSpacing = 16.0
+    static let titleLabelMargin = 8.0
+    static let titleLabelTopMargin = 20.0
+    static let headerStackViewMargin = 8.0
+    static let headerStackViewTopMargin = 20.0
+    static let tableViewSpacerTopMargin = 20.0
+    static let tableViewSpacerHeight = 1.0
+    static let tableViewTopMargin = 20.0
+}
+
+typealias CertificateItems = [(key: String, value: [String])]
+
+class CertificatesViewController: UIViewController, Themeable, UITableViewDelegate, UITableViewDataSource {
+    // MARK: - UI
+    private let titleLabel: UILabel = .build { label in
+        label.font = FXFontStyles.Bold.title1.scaledFont()
+        label.text = .Menu.EnhancedTrackingProtection.certificatesTitle
+    }
+
+    private let headerStackView: UIStackView = .build { stack in
+        stack.axis = .horizontal
+        stack.distribution = .fillEqually
+        stack.spacing = CertificatesUX.headerStackViewSpacing
+        stack.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
One of the things the `.build` helper does is set `translatesAutoresizingMaskIntoConstraints` to `false` so that's probably unnecessary in these view/constraint setups (also lines 54).

Also, if you use the `.build` setup for the `certificatesTableView`, you could put some of the view setup there and save `setupCertificatesTableView` just for constraints. And then you also wouldn't need line 154.